tests/testthat/setup.R

```diff
@@ -56,6 +56,9 @@ renv_tests_init_options <- function() {
 renv_tests_init_packages <- function() {
   requireNamespace("waldo", quietly = TRUE)
   renv_namespace_load("crayon")
+  renv_namespace_load("diffobj")
```

Comment:
I think I should probably switch back to the previous approach; but maybe I've got all the important deps now?

Reply:
It might be worthwhile just in case the dependencies (or recursive dependencies?) of these packages change in a way that break `renv` unit tests.